src/test/java/net/openhft/chronicle/queue/MappedFileUtil.java

```diff
@@ -0,0 +1,66 @@
+package net.openhft.chronicle.queue;
+
+import net.openhft.chronicle.core.Jvm;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public enum MappedFileUtil {
+    ;
+    private static final Path PROC_SELF_MAPS = Paths.get("/proc/self/maps");
+
+    // See https://man7.org/linux/man-pages/man5/proc.5.html
+    private static final Pattern LINE_PATTERN = Pattern.compile("([A-Fa-f\\d\\-]+)\\s+([rwxsp\\-]+)\\s+([A-Fa-f\\d]+)\\s+([A-Fa-f\\d]{2}:[A-Fa-f\\d]{2})\\s+(\\d+)\\s*(.*)?");
```

Comment:
phew!

Reply:
I love a good regex. A lot of the bulk is the hexadecimal number pattern repeated `[A-Fa-f\d]+`. I swapped it out for the `\p{XDigit}` macro, hopefully that makes it less of a sledge hammer in the face.